docs/reference/index.md

```diff
@@ -1,17 +1,101 @@
 # Reference
 
+These guides provide technical information about Pebble.
+
+
+## Layers
+
+Pebble configuration is defined as a stack of "layers".
+
 ```{toctree}
 :titlesonly:
 :maxdepth: 1
 
-Changes and tasks <changes-and-tasks>
-CLI Commands <cli-commands/cli-commands>
-Health checks <health-checks>
-Identities <identities>
 Layers <layers>
 Layer specification <layer-specification>
-Log forwarding <log-forwarding>
-Notices <notices>
-Service auto-restart <service-auto-restart>
+```
+
+
+## Pebble commands
+
+The `pebble` command has several subcommands.
+
+```{toctree}
+:titlesonly:
+:maxdepth: 1
+
+CLI Commands <cli-commands/cli-commands>
+```
+
+
+## Pebble in containers
+
+When Pebble is configured as a client connected to a remote system (for example, a separate container), you can use subcommands on the client to manage the remote system.
```

Comment:
I find this sentence slightly inaccurate because Pebble can't connect to a container.

How about being more precise here, something like "When the Pebble daemon is running inside a remote system (for example, a separate container), the Pebble client subcommands can be used to manage the remote system."? Maybe you can come up with something better than this :)

Reply:
Good point, thanks, I think this is a better approach. I've added https://github.com/canonical/pebble/pull/524/commits/7ab69e1e37c015c9f1c4018a7713b1b47bb17c0a with new wording